code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -624,13 +624,18 @@
 	desc = "A special box for sensitive people."
 	icon_state = "hugbox"
 	foldable = null
+	var/cooldown = 0
 
 /obj/item/weapon/storage/box/hug/attack_self(mob/user)
-	..()
-	user.changeNext_move(CLICK_CD_MELEE)

```

Comment:
Because I had no idea what that did, and just left it there because I
assumed it was needed.

I'll do that, thanks.